docs/developer.md

```diff
@@ -211,17 +216,27 @@ For example, if we add a new feature or make an update to an existing UI / user
 anything that breaks functionality for users or systems that depend on TermHub, that would be a `MINOR` version update.
 So if the version was 1.10.2 before, it would be 1.11.0 after.
 
-#### How to do a version update
+### How to do a version update
 1. Update the version in `frontend/src/env.js`
 2. Tag the version in GitHub: `git tag VERSION; git push --tags`
 
-### Periodic maintenance
-#### Updating auth token
+## Periodic maintenance
+### Updating environmental variables
```

Comment:
## Update: docs
Updated env information, particularly regarding oauth / service token, and added new section "Updating environmental variables".